src/server/ApiManager.js

```diff
@@ -0,0 +1,96 @@
+class ApiManager {
+
+
+    /* Bike Shop API */
+    /*
+    constructor() {
+
+        this.apiUrlEndpoint = '/bike-shop-api/';
+
+        this.apiUrlMap = new Map();
+        this.apiUrlMap.set('high-all', 'high-low-all-products');
+        this.apiUrlMap.set('low-all', 'low-high-all-products');
+        this.apiUrlMap.set('high-accessories', 'high-low-accessories-products');
+        this.apiUrlMap.set('low-accessories', 'low-high-accessories-products');
+        this.apiUrlMap.set('high-bikes', 'high-low-bikes-products');
+        this.apiUrlMap.set('low-bikes', 'low-high-bikes-products');
+        this.apiUrlMap.set('high-components', 'high-low-components-products');
+        this.apiUrlMap.set('low-components', 'low-high-components-products');
+    }
+    
+    getCategoryUrl(categoryId, sort) {
+
+        var apiUrlKey = sort + '-' + categoryId;
+        return this.apiUrlEndpoint + this.apiUrlMap.get(apiUrlKey) + '.json';
+
+    }
+    */
+
+    /* Campmor API */
+    constructor() {
+
+        this.apiUrlEndpoint = 'https://campmor.ampify.wompmobile.com/campmor';
+        this.apiCategoriesEndpoint = this.apiUrlEndpoint + '/fetchCategories';
+        this.apiProductEndpoint = this.apiUrlEndpoint + '/fetchProduct';
+
+        this.apiUrlMap = new Map();
```

Comment:
Do you mind if we do this differently?  Instead of hardcoding certain types of searches to certain Womp API params , we could have:

- a mapping from category name to category ID  (e.g., men-shirts to 200368502)
- if necessary, a mapping from high-low to priceHtoL and low-high to priceLtoH.  Though I'm also personally fine with just using priceHtoL and priceLtoH in our app.

You could then generate API params this way.

Reply:
Ben! What if we remove completely this map, and  send params directly from the page?
I initially created it, while we were deciding which API to use, in order to eventually change the product API with minimal impact on the client, but it doesn't seem to make sense anymore. 
Thoughts?